ngi_pipeline/engines/piper_ngi/parsers.py

```diff
@@ -155,3 +155,20 @@ def parse_genotype_concordance(genotype_concordance_file):
                                        gt_entry.overall_genotype_concordance))
             continue
     return samples_gtc_dict
+
+def parse_deduplication_percentage(deduplication_file):
+
+    duplication_percentage=0
+    with open(deduplication_file, 'r') as f:
+        for line in iter(f.readline, ''):
+            if "## METRICS CLASS" in line and "picard.sam.DuplicationMetrics" in line:
```

Comment:
this is really a matter of taste... but I would prefer to capture the header line using the fact the it "must" start with `LIBRARY` rather than assuming that after the comment `## METRICS CLASS` there must be the header.
As far as I know this is always true, however a picard metric file that has another comment after the `## METRICS CLASS` comment is perfectly valid from a practical point of view.